### ISSUE TITLE
Specify remaining required/optional fields

### ISSUE THREAD

**Issue by danieldegroot2** [thomersch/openstreetmap-calendar#94]
Specify which fields are required and which are optional. Currently only End time is displayed as optional, while some other fields are also optional (Link, Location name, Description, Sign Up Survey).

**Comment by thomersch** [comment]
That's right: The UI is not explicit about it. The reasoning is that I want to encourage users to fill as much as possible, because more in-place information increases user value, e.g. a description massively improves the usefulness for someone who just got a link. I am afraid that explicitly marking such fields as optionals, it might make users think that those are not important fields which can be skipped.
